crates/hdk3/Cargo.toml

```diff
@@ -20,7 +20,7 @@ holo_hash = { path = "../holo_hash" }
 holochain_wasmer_guest = "=0.0.66"
 holochain_zome_types = { path = "../holochain_zome_types" }
 paste = "=1.0.3"
-serde = "1.0.104"
+serde = "1.0"
```

Comment:
given that we lock to serialized bytes and that locks to a serde version, this could probably just be `"1"` even

Reply:
Kept it 1.0 for now, there are lots of constraints we could loosen